pkg/cmd/cli/describe/printer.go

```diff
@@ -996,8 +996,12 @@ func printNetNamespaceList(list *sdnapi.NetNamespaceList, w io.Writer, opts kpri
 }
 
 func printClusterNetwork(n *sdnapi.ClusterNetwork, w io.Writer, opts kprinters.PrintOptions) error {
+	var cidrList []string
+	for _, cidr := range n.ClusterDef {
+		cidrList = append(cidrList, cidr.ClusterNetworkCIDR)
+	}
 	name := formatResourceName(opts.Kind, n.Name, opts.WithKind)
-	_, err := fmt.Fprintf(w, "%s\t%s\t%d\t%s\t%s\n", name, n.Network, n.HostSubnetLength, n.ServiceNetwork, n.PluginName)
+	_, err := fmt.Fprintf(w, "%s\t%s\t%d\t%s\t%s\n", name, strings.Join(cidrList, ","), n.HostSubnetLength, n.ServiceNetwork, n.PluginName)
```

Comment:
Yes, it will hard to read when there are several cidrs. May be one row for each cidr?
```
for _, cidr := range n.ClusterDef {
   , err := fmt.Fprintf(w, "%s\t%s\t%d\t%s\t%s\n", name, cidr. ...)
}
```